packages/application-shell-connectors/src/utils/get-mc-api-url/get-mc-api-url.ts

```diff
@@ -1,5 +1,8 @@
 import type { ApplicationWindow } from '@commercetools-frontend/constants';
 
+const removeMcPrefix = (host: string) =>
+  host.replace(/^mc(-(\d){4,})?\.(.*)$/, '$3');
```

Comment:
This now includes our preview URLs, so there is no need to have the `disableInferringOfMcApiUrlOnProduction` option anymore.